gnosis/eth/tests/ethereum_test_case.py

```diff
@@ -8,22 +8,38 @@
 from web3.contract import Contract
 
 from ..ethereum_client import EthereumClient, EthereumClientProvider
+from ..multicall import Multicall
 from .utils import deploy_erc20, deploy_example_erc20, send_tx
 
 logger = logging.getLogger(__name__)
 
 
+_cached_data = {
+    'ethereum_client': None,  # Prevents initializing again
+}
+
+
 class EthereumTestCaseMixin:
-    ethereum_client: EthereumClient
-    w3: Web3
-    ethereum_test_account: LocalAccount
+    ethereum_client: EthereumClient = None
+    w3: Web3 = None
+    ethereum_test_account: LocalAccount = None
+    multicall: Multicall = None
 
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
```

Comment:
⚠️ There's no `super()` here to be called 🤔

Reply:
Well, that's because it's a `mixin` and not a "proper" class. It's intended to be inherited with another class (a `TestCase` in this case)

Reply:
Got it 🙌